lib/mix/lib/mix/tasks/profile.fprof.ex

```diff
@@ -0,0 +1,326 @@
+defmodule Mix.Tasks.Profile.Fprof do
+  use Mix.Task
+
+  @shortdoc "Profiles the given file or expression with fprof"
+
+  @moduledoc """
+  Profiles the given file or expression using Erlang's `fprof` tool.
+
+  `fprof` can be useful when you want to discover the bottlenecks of a
+  sequential code.
+
+  Before running the code, it invokes the `app.start` task which compiles
+  and loads your project. Then the target expression is profiled, together
+  with all processes which are spawned by it. Other processes (e.g. those
+  residing in the OTP application supervision tree) are not profiled.
+
+  To profile the code, you can use the syntax similar to `mix run` task:
+      mix profile.fprof -e Hello.world
+      mix profile.fprof my_script.exs arg1 arg2 arg3
+
+  ## Command line options
+
+    * `--callers`       - print detailed information about immediate callers and called functions
+    * `--details`       - include profile data for each profiled process
+    * `--sort key`      - sort the output by given key: acc (default) or own
+    * `--config`, `-c`  - loads the given configuration file
+    * `--eval`, `-e`    - evaluate the given code
+    * `--require`, `-r` - require pattern before running the command
+    * `--parallel-require`, `-pr` - requires pattern in parallel
+    * `--no-compile`    - do not compile even if files require compilation
+    * `--no-deps-check` - do not check dependencies
+    * `--no-start`      - do not start applications after compilation
+
+  ## Profile output
+
+  The example output looks as following:
+                                               CNT    ACC (ms)    OWN (ms)
```

Comment:
A table is an interesting workaround, although I'd prefer to have it as a standard code snippet (`pre` element) since this is the output of the task. But yeah, that might be one way of dealing with it. I could say, "The task produces the following table:" or something similar.